blurring_as_a_service/metrics/total_blurred_area.py

```diff
@@ -0,0 +1,57 @@
+import numpy as np
+
+
+class TotalBlurredArea:
+    def __init__(self):
+        self.f1_score = None
+        self.recall = None
+        self.precision = None
+        self.tp = 0
+        self.fp = 0
+        self.tn = 0
+        self.fn = 0
+
+    def update_statistics_based_on_masks(self, true_mask, predicted_mask):
+        """
+        Computes statistics for a given pair of binary masks.
+        Args:
+            true_mask: numpy array of shape (height, width)
+            predicted_mask: numpy array of shape (height, width)
+
+        Returns:
+
+        """ """
+        Computes true positives and false positives for a given pair of binary masks.
+        :param true_mask: numpy array of shape (height, width)
+        :param pred_mask: numpy array of shape (height, width)
+        """
+        tp = np.sum((true_mask == 1) & (predicted_mask == 1))
```

Comment:
Can this two blocks be simplified in one?
For example:
`self.tp += np.sum((true_mask == 1) & (predicted_mask == 1))`